packages/fakta/omsorg-og-rett/src/components/OmsorgOgRettInfoPanel.tsx

```diff
@@ -13,53 +13,95 @@ import { OpplysningerFraSoknad } from '../opplysningskort/OpplysningerFraSoknad.
 import { OpplysningerOmAdresser } from '../opplysningskort/OpplysningerOmAdresser.tsx';
 import { AleneomsorgForm } from './forms/AleneomsorgForm';
 import { HarAnnenForelderRettForm } from './forms/HarAnnenForelderRettForm';
+import { RettighetstypeForm } from './forms/RettighetstypeForm.tsx';
 
-export const OmsorgOgRettInfoPanel = ({ personoversikt, omsorgOgRett, submittable }: OmsorgOgRettProps) => {
+export const OmsorgOgRettInfoPanel = ({
+  personoversikt,
+  omsorgOgRett,
+  submittable,
+  kanOverstyre,
+}: OmsorgOgRettProps) => {
   const { alleKodeverk, aksjonspunkterForPanel, isReadOnly, harÅpneAksjonspunkter } = usePanelDataContext();
 
-  const harAPAleneomsorg = hasAksjonspunkt(
-    AksjonspunktKode.MANUELL_KONTROLL_AV_OM_BRUKER_HAR_ALENEOMSORG,
-    aksjonspunkterForPanel,
+  const aksjonspunkter = aksjonspunkterForPanel.filter(
+    a => a.definisjon !== AksjonspunktKode.OVERSTYRING_AV_RETT_OG_OMSORG,
   );
-  const harAPAnnenForelderRett = hasAksjonspunkt(AksjonspunktKode.AVKLAR_ANNEN_FORELDER_RETT, aksjonspunkterForPanel);
+
+  const opprettetAleneomsorgAPUtenResultat = aksjonspunkter.some(
+    a =>
+      a.definisjon == AksjonspunktKode.MANUELL_KONTROLL_AV_OM_BRUKER_HAR_ALENEOMSORG &&
```

Comment:
Bør bruka === på alle desse nedover. Det spelar ingen rolle her, sidan ein samanlignar strings. Men for ordens skyld brukar me alltid ==. https://learn.onemonth.com/difference-between-equal-signs-javascript/